resources/options_lookup.tsv

```diff
@@ -483,6 +483,7 @@ Water Heater Fuel	Other Fuel
 Water Heater Fuel	Propane																										
 Water Heater Efficiency	Electric Standard	ResStockArguments	water_heater_type=storage water heater	water_heater_fuel_type=electricity	water_heater_location=auto	water_heater_tank_volume=auto	water_heater_efficiency_type=EnergyFactor	water_heater_efficiency=0.92	water_heater_recovery_efficiency=0	water_heater_standby_loss=0	water_heater_jacket_rvalue=0	water_heater_setpoint_temperature=125	water_heater_heating_capacity=auto	water_heater_has_flue_or_chimney=auto	water_heater_num_units_served=1												
 Water Heater Efficiency	Electric Premium	ResStockArguments	water_heater_type=storage water heater	water_heater_fuel_type=electricity	water_heater_location=auto	water_heater_tank_volume=auto	water_heater_efficiency_type=EnergyFactor	water_heater_efficiency=0.95	water_heater_recovery_efficiency=0	water_heater_standby_loss=0	water_heater_jacket_rvalue=0	water_heater_setpoint_temperature=125	water_heater_heating_capacity=auto	water_heater_has_flue_or_chimney=auto	water_heater_num_units_served=1												
+Water Heater Efficiency	Electric Tankless Overaged	ResStockArguments	water_heater_type=instantaneous water heater	water_heater_fuel_type=electricity	water_heater_location=auto	water_heater_tank_volume=0	water_heater_efficiency_type=EnergyFactor	water_heater_efficiency=0.99	water_heater_recovery_efficiency=0	water_heater_standby_loss=0	water_heater_jacket_rvalue=0	water_heater_setpoint_temperature=125	water_heater_heating_capacity=auto	water_heater_has_flue_or_chimney=auto	water_heater_num_units_served=1												

```

Comment:
Duplicated the Electric Tankless with a new name as a quick workaround to NOT model a HPWH upgrade in case the Electric Tankless is < 10 years